portal-screencast/src/lib.rs

```diff
@@ -262,29 +276,76 @@ impl std::ops::Drop for ActiveScreenCast {
 #[derive(Debug)]
 pub struct ScreenCastStream {
     pipewire_node: u32,
-    // TODO: other stream metadata.
+    width: u32,
+    height: u32,
 }
 
 impl ScreenCastStream {
     /// Get the PipeWire node ID for this stream.
     pub fn pipewire_node(&self) -> u32 {
         self.pipewire_node
     }
+
+    pub fn width(&self) -> u32 {
+        self.width
+    }
+
+    pub fn height(&self) -> u32 {
+        self.height
+    }
+
+    pub fn size(&self) -> (u32, u32) {
+        (self.width(), self.height())
+    }
```

Comment:
Not sure if we want all 3 maybe Size is enough -- im open to suggestions

Reply:
I can see the idividual accessors being useful so I guess this does no harm. I'm happy to accept it.